assets/script.js

```diff
@@ -12,15 +12,15 @@ var Spoonacular_API = null;
 //Recipe Request Page DOM
 const searchBtn = document.querySelector("#search");
 const saveBtn = document.querySelector("#save");
-const nextBtn = Array.from(document.querySelectorAll(".nav-btn-inline"));
+const nextBtn = document.querySelector("#next");
 const resultContainer = document.querySelector("#results-container");
 const recipeNavBtns = document.querySelector("#btn-row");
 //API URLs
 const fetchhRecipesURL = `https://api.spoonacular.com/recipes/complexSearch`;
 const fetchCaloriesBurnt = `https://api.api-ninjas.com/v1/caloriesburnedactivities`;
 const fetchExercises = `https://api.api-ninjas.com/v1/exercises?`;
 
-//exporting information
+//exporting information -Keiji what is this for? do you need t
```

Comment:
My bad, that was me earlier trying to get a code to work. I realized I was approaching it wrong. I will get rid of it for my next push.